src/redux/book/booksSlice.js

```diff
@@ -1,46 +1,43 @@
 /* eslint-disable no-param-reassign */
```

Comment:
- [ ] @ismailmunyentwari9 It's not good practice to disable the linter check in your file, kindly remove this line and fix the linter errors instead of disabling the check.